main.tex

```diff
@@ -1033,9 +1033,8 @@ \subsubsection{Plotting image data with world coordinates}
 coordinate grid. The challenge, however, is that the conceptual coordinate axes
 (such as longitude/latitude) need not be lined up with the pixel axes of the
 image. The \package{astropy.visualization.wcsaxes} subpackage implements a
-generalized way of making figures from an image array and a world coordinate
-system (WCS) object that provides the transformation between pixel and
-world coordinates.
+generalized way of making figures from an image array and a WCS object
```

Comment:
Note: WCS acronym is already introduced earlier in the paper, so we can just reap the benefits here.